docs/delorean/deploy-jenkins.md

```diff
@@ -0,0 +1,18 @@
+# Deploy Jenkins
+
+## Table of Contents
+  - [1. Prerequisites](#prerequisites)
+  - [2. Deploy Jenkins on OpenShift](#deploy-on-openshift)
+
+
+## Prerequisites
+- OpenShift 3.x Cluster (oc login)
+
+## Deploy on OpenShift
```

Comment:
Can make this the same as the heading in the table of contents? `Deploy Jenkins on OpenShift`

Reply:
Matches now